src/command_modules/azure-cli-lab/azure/cli/command_modules/lab/validators.py

```diff
@@ -114,6 +114,12 @@ def validate_template_id(namespace):
                                              grandchild_name=namespace.arm_template)
 
 
+def validate_claim_vm(namespace):
+    if namespace.name is None and namespace.lab_name is None or namespace.resource_group is None:
+        raise CLIError("usage error: [--ids IDs] | [--lab-name LabName --resource-group ResourceGroup --name VMName]"
+                       " | [--lab-name LabName --resource-group ResourceGroup]")
```

Comment:
This implies that all of these sets are optional and thus that the command should work without any of them, which I don't think is the case.  I'd recommend dropping the [ ] around the sets.